.husky/pre-commit

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
+# check typos
+typos .
```

Comment:
How is the `typos` tool installed? Do we need to add something to our README.md?

Reply:
[725a2ff](https://github.com/Snowfork/snowbridge/pull/747/commits/725a2ff0ab5dcb543736fe91e5cdd52bd19dedd2)